.github/pipelines.sh

```diff
@@ -3,4 +3,5 @@ set -ex
 
 for i in $(ls config); do 
     gomplate --left-delim "{{{" --right-delim "}}}" -V --datasource config=config/$i --file build.yaml.gomplate > workflows/build-$i
+    sed -i '/^[[:space:]]*$/d' workflows/build-$i
```

Comment:
I think mostly comes from templates block not having `{{-` prepended, so this shouldn't be really required.. but if it works fine by me :)